provider/ecs/ecs.go

```diff
@@ -469,6 +470,14 @@ func (p *Provider) getFrontendRule(i ecsInstance) string {
 	return "Host:" + strings.ToLower(strings.Replace(i.Name, "_", "-", -1)) + "." + p.Domain
 }
 
+func (p *Provider) getBasicAuth(i ecsInstance) []string {
+	label := i.label(types.LabelFrontendAuthBasic)
+	if label != "" {
+		return strings.Split(label, ",")
```

Comment:
Do we need/also want to check that each basic auth value adheres to the `user:hash`?

Reply:
@ldez This should have been answered prior to merging.

Reply:
https://github.com/containous/traefik/blob/master/provider/docker/docker.go#L706